src/components/TodoList.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+import { TodoInfo } from './TodoInfo';
+
+type Props = {
+  todos: Todo[],
+  temporaryTodo: Todo | null,
+  onDelete: (id: number) => void,
+  onToggle: (id: number) => void,
+  loadingTodoIds: number[],
+};
+
+export const TodoList: React.FC<Props> = ({
+  todos, temporaryTodo, onDelete, onToggle, loadingTodoIds,
+}) => {
+  return (
+    <section className="todoapp__main">
```

Comment:
It's recommended to use the 'ul' element to represent a list of items instead of 'section'. Wrap the 'TodoInfo' components with the 'ul' and 'li' elements for better semantics and accessibility.

Reply:
changed section on ul